### DISCUSSION TITLE
[Support]: latest DEV version issue (0.15.0-959ca0f)

### DISCUSSION THREAD

**Discussion by a-lunatic** [blakeblackshear/frigate#14775] [Ask A Question] [answered]
Trying out the latest dev version I started getting this in the logs and don't use OpenVINO only use USB coral and nvidia for ffmpeg hwaccel_args and now it locks up on first detection even after trying the 15 beta_1 locks it up but doesn't have this error.

2024-11-04 18:54:47.958517322  Process embeddings_manager:
2024-11-04 18:54:47.958521803  Traceback (most recent call last):
2024-11-04 18:54:47.958524150    File "/usr/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
2024-11-04 18:54:47.958529086      self.run()
2024-11-04 18:54:47.958532155    File "/opt/frigate/frigate/util/process.py", line 41, in run_wrapper
2024-11-04 18:54:47.958534181      return run(*args, **kwargs)
2024-11-04 18:54:47.958563396    File "/usr/lib/python3.9/multiprocessing/process.py", line 108, in run
2024-11-04 18:54:47.958565984      self._target(*self._args, **self._kwargs)
2024-11-04 18:54:47.958568895    File "/opt/frigate/frigate/embeddings/__init__.py", line 59, in manage_embeddings
2024-11-04 18:54:47.958570891      maintainer = EmbeddingMaintainer(
2024-11-04 18:54:47.958572913    File "/opt/frigate/frigate/embeddings/maintainer.py", line 48, in __init__
2024-11-04 18:54:47.958614320      self.embeddings = Embeddings(config.semantic_search, db)
2024-11-04 18:54:47.958616370    File "/opt/frigate/frigate/embeddings/embeddings.py", line 114, in __init__
2024-11-04 18:54:47.958618178      self.vision_embedding = GenericONNXEmbedding(
2024-11-04 18:54:47.958633553    File "/opt/frigate/frigate/embeddings/functions/onnx.py", line 90, in __init__
2024-11-04 18:54:47.958653984      self._load_model_and_tokenizer()
2024-11-04 18:54:47.958655665    File "/opt/frigate/frigate/embeddings/functions/onnx.py", line 136, in _load_model_and_tokenizer
2024-11-04 18:54:47.958656927      self.runner = ONNXModelRunner(
2024-11-04 18:54:47.958658232    File "/opt/frigate/frigate/util/model.py", line 100, in __init__
2024-11-04 18:54:47.958659418      self.interpreter = self.ov.compile_model(
2024-11-04 18:54:47.958661152    File "/usr/local/lib/python3.9/dist-packages/openvino/runtime/ie_api.py", line 543, in compile_model
2024-11-04 18:54:47.958662468      super().compile_model(model, device_name, {} if config is None else config),
2024-11-04 18:54:47.958678815  RuntimeError: Exception from src/inference/src/cpp/core.cpp:121:
2024-11-04 18:54:47.958680205  Check '!m_device_map.empty()' failed at src/plugins/intel_gpu/src/plugin/plugin.cpp:379:
2024-11-04 18:54:47.958681669  [GPU] Can't get OPTIMIZATION_CAPABILITIES property as no supported devices found or an error happened during devices query.
2024-11-04 18:54:47.958683361  [GPU] Please check OpenVINO documentation for GPU drivers setup guide.

**Comment by hawkeye217** [comment]
See the docs: https://deploy-preview-13787--frigate-docs.netlify.app/configuration/semantic_search#gpu-acceleration

If you are trying to use the new semantic search feature with the large model, you need to download and use the Docker image specific to your GPU. Setting model_size to small will run semantic search on your CPU and you won't need to download a new image.

**Reply by a-lunatic** [reply]
> If you are trying to use the new semantic search feature with the large model, you need to download and use the Docker image specific to your GPU. Setting model_size to small will run semantic search on your CPU and you won't need to download a new image.

I had the semantic search feature with the large model working with 0.15.0 e5ebf93-amd64 which is running now without issues.

The next version I tried was 189d4b4-amd64 & 959ca0f-amd64 which give this error.

**Reply by NickM-27** [reply]
I know what the issue is, will fix it

**Reply by hawkeye217** [reply]
Some changes have been committed in https://github.com/blakeblackshear/frigate/pull/14786, can you see if this resolves your issue?

**Reply by a-lunatic** [reply]
a13b981-amd64 still locks up on the first detection live views still work and this is the error now

[2024-11-04 22:22:17] frigate.util.model             WARNING : OpenVINO failed to build model, using CPU instead: Exception from src/inference/src/cpp/core.cpp:121:
2024-11-04 22:22:17.500415201  Check '!m_device_map.empty()' failed at src/plugins/intel_gpu/src/plugin/plugin.cpp:379:
2024-11-04 22:22:17.500418505  [GPU] Can't get OPTIMIZATION_CAPABILITIES property as no supported devices found or an error happened during devices query.
2024-11-04 22:22:17.500421537  [GPU] Please check OpenVINO documentation for GPU drivers setup guide.

**Reply by NickM-27** [reply]
in what way does it lock up?

**Reply by a-lunatic** [reply]
Birdseye view is frozen and camera views and debug view show no frames but when clicking on a camera from live view they still show live video.

**Reply by a-lunatic** [reply]
Going back to 0.15.0-e5ebf93 makes it lock up in the same way too have remove the 0.15.0 stuff from config load 0.14.1 start it then load  e5ebf93-amd64 to get it work again.

**Reply by hawkeye217** [reply]
This almost makes it seem like it's some kind of docker cache issue.

**Reply by hawkeye217** [reply]
Or, intel updated their packages and broke something...

**Reply by a-lunatic** [reply]
Yeah it's got me stuffed it also fires an HA automation for person detection on start up of frigate which started from 189d4b4
-amd64 .

e5ebf93-amd64 is working fine after going back to 0.14.0 then updating.

**Reply by hawkeye217** [reply]
> Yeah it's got me stuffed it also fires an HA automation for person detection on start up of frigate which started from [189d4b4](https://github.com/blakeblackshear/frigate/commit/189d4b459fa83f93b782e4a4f00f039ba76d3371)
> -amd64 .

Firing an automation seems entirely unrelated to that commit.

**Reply by a-lunatic** [reply]
Latest dev version 0.15.0 fc0fb15-amd64 is working now with large model.

Tried to get the fc0fb15-tensorrt image going but doesn't work at all this is what the logs show.

s6-rc: info: service s6rc-fdholder: starting
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service s6rc-fdholder successfully started
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service trt-model-prepare: starting
s6-rc: info: service log-prepare: starting
Generating the following TRT Models: yolov7-320
Downloading yolo weights
s6-rc: info: service log-prepare successfully started
s6-rc: info: service nginx-log: starting
s6-rc: info: service go2rtc-log: starting
s6-rc: info: service frigate-log: starting
s6-rc: info: service certsync-log: starting
s6-rc: info: service nginx-log successfully started
s6-rc: info: service go2rtc-log successfully started
s6-rc: info: service go2rtc: starting
s6-rc: info: service frigate-log successfully started
s6-rc: info: service certsync-log successfully started
s6-rc: info: service go2rtc successfully started
s6-rc: info: service go2rtc-healthcheck: starting
s6-rc: info: service go2rtc-healthcheck successfully started
2024-11-06 14:04:17.471679845  [INFO] Preparing new go2rtc config...
2024-11-06 14:04:17.916987424  [INFO] Not injecting WebRTC candidates into go2rtc config as it has been set manually
2024-11-06 14:04:17.963929543  [INFO] Starting go2rtc...
2024-11-06 14:04:18.092059240  14:04:18.091 INF go2rtc platform=linux/amd64 revision=b2399f3 version=1.9.2
2024-11-06 14:04:18.092118872  14:04:18.092 DBG build vcs.time=2024-05-17T12:57:11Z version=go1.22.2
2024-11-06 14:04:18.092120912  14:04:18.092 INF config path=/dev/shm/go2rtc.yaml
2024-11-06 14:04:18.092758013  14:04:18.092 INF [api] listen addr=:1984
2024-11-06 14:04:18.093021973  14:04:18.092 INF [rtsp] listen addr=:8554
2024-11-06 14:04:18.093547136  14:04:18.093 INF [webrtc] listen addr=:8555
2024-11-06 14:04:18.093923427  14:04:18.093 DBG [hass] can't import config: open /config/.storage/core.config_entries: no such file or directory
2024-11-06 14:04:19.908450324  14:04:19.908 DBG [rtsp] new consumer stream=birdseye
2024-11-06 14:04:19.908778662  14:04:19.908 DBG [exec] run cmd="/usr/lib/ffmpeg/7.0/bin/ffmpeg -hide_banner -f rawvideo -pix_fmt yuv420p -video_size 2560x1440 -r 10 -i /tmp/cache/birdseye -c:v h264_nvenc -g 50 -profile:v high -level:v auto -preset:v p2 -tune:v ll -rtsp_transport tcp -f rtsp rtsp://127.0.0.1:8554/74c7b74f122b910645740cb9fdeb77a9" url="exec:/usr/lib/ffmpeg/7.0/bin/ffmpeg -hide_banner -f rawvideo -pix_fmt yuv420p -video_size 2560x1440 -r 10 -i /tmp/cache/birdseye -c:v h264_nvenc -g 50 -profile:v high -level:v auto -preset:v p2 -tune:v ll -rtsp_transport tcp -f rtsp rtsp://127.0.0.1:8554/74c7b74f122b910645740cb9fdeb77a9"
2024-11-06 14:04:19.938820366  [in#0 @ 0x61401a934d40] Error opening input: No such file or directory
2024-11-06 14:04:19.938835128  Error opening input file /tmp/cache/birdseye.
2024-11-06 14:04:19.938838488  Error opening input files: No such file or directory
2024-11-06 14:04:19.941141036  14:04:19.940 WRN [rtsp] error="streams: exec: [in#0 @ 0x61401a934d40] Error opening input: No such file or directory\nError opening input file /tmp/cache/birdseye.\nError opening input files: No such file or directory\n" stream=birdseye
2024-11-06 14:04:27.473497900  [INFO] Starting go2rtc healthcheck service...

**Reply by NickM-27** [reply]
What makes you think it's not working?

**Reply by a-lunatic** [reply]
Web UI doesn't load at all and that's all the log prints out.

**Reply by NickM-27** [reply]
Because it's still generating the trt model

```
Generating the following TRT Models: yolov7-320
Downloading yolo weights
```

**Reply by a-lunatic** [reply]
How large is it as it has been going for 15 mins still nothing.

This is in the log after running for ~2 mins
s6-rc: warning: unable to start service trt-model-prepare: command exited 4

**Reply by NickM-27** [reply]
looks like there is an issue, but in any case we might want to make it not build any models by default since users will be using this for things other than detection now

**Comment by jurasjo** [comment]
Looks like it is similar problem to what I have been observing, but in my case I do use OpenVino. 
Look here: [https://github.com/blakeblackshear/frigate/pull/14728](https://github.com/blakeblackshear/frigate/pull/14728)

The last commit I can run is: frigate:11068aa-amd64

I will try to do some debugging on my own and if I can't find anything, I will create and issue.

**Reply by hawkeye217** [reply]
Your issue is different than the OP's. The stack trace is different. As I suggested on the PR, go ahead and open your own support discussion so we don't mix issues.

**Comment by coldkevin** [comment]
im having the exact same issue as OP, but reading this im also not using the tensor image. Using coral for detection and nvidia gpu for acceleration. Would me switching to the tensor image cause any issues with my current setup? can i still use the coral as a detector? @hawkeye217

**Reply by NickM-27** [reply]
you can still use the coral for detection, if you want to use nvidia gpu for the semantic search model then you must use the tensorrt image. What is your current config?

**Reply by coldkevin** [reply]
```
mqtt:
  host: 192.168.1.34
  port: 1883
  topic_prefix: frigate
  client_id: frigate
  user: user
  password: password
  stats_interval: 60

auth:
  enabled: true

tls:
  enabled: false

detectors:
  coral_pci:
    type: edgetpu
    device: pci

model:
  path: plus://ed6b9450b4c391f6951772ff2b2196e0

logger:
  # Optional: Default log verbosity (default: shown below)
  default: warning
  # Optional: Component specific logger overrides
  logs:
    frigate.event: debug

record:
  sync_recordings: true
  
genai:
  enabled: true
  provider: gemini
  api_key: APIKEY
  model: gemini-1.5-flash

semantic_search:
  enabled: true
  model_size: large  
  reindex: true
  
ffmpeg:
  # Optional: global ffmpeg args (default: shown below)
  global_args: -hide_banner -loglevel warning -threads 2
  # Optional: global hwaccel args (default: shown below)
  # NOTE: See hardware acceleration docs for your specific device
  hwaccel_args: preset-nvidia-h264
  # Optional: global input args (default: shown below)
  input_args: preset-rtsp-generic
  # Optional: global output args
  output_args:
    # Optional: output args for detect streams (default: shown below)
    detect: -threads 2 -f rawvideo -pix_fmt yuv420p
    # Optional: output args for record streams (default: shown below)
    record: preset-record-generic-audio-aac
    # Optional: output args for rtmp streams (default: shown below)
  retry_interval: 25

birdseye:
  # Optional: Enable birdseye view (default: shown below)
  enabled: false
  # Optional: Restream birdseye via RTSP (default: shown below)
  # NOTE: Enabling this will set birdseye to run 24/7 which may increase CPU usage somewhat.
  restream: false
  # Optional: Width of the output resolution (default: shown below)
  width: 1280
  # Optional: Height of the output resolution (default: shown below)
  height: 720
  # Optional: Encoding quality of the mpeg1 feed (default: shown below)
  # 1 is the highest quality, and 31 is the lowest. Lower quality feeds utilize less CPU resources.
  quality: 8
  # Optional: Mode of the view. Available options are: objects, motion, and continuous
  #   objects - cameras are included if they have had a tracked object within the last 30 seconds
  #   motion - cameras are included if motion was detected in the last 30 seconds
  #   continuous - all cameras are included always
  mode: continuous
  # Optional: Threshold for camera activity to stop showing camera (default: shown below)
  inactivity_threshold: 30
  # Optional: Configure the birdseye layout
  layout:
    # Optional: Scaling factor for the layout calculator (default: shown below)
    scaling_factor: 2.0
    # Optional: Maximum number of cameras to show at one time, showing the most recent (default: show all cameras)
    max_cameras: 5

go2rtc:
  streams:
    2way_driveway:
      - rtsp://user:pass@192.168.1.101:554/cam/realmonitor?channel=1&subtype=0
      - ffmpeg:Driveway#audio=opus
    2way_driveway_sub:
      - rtsp://user:pass@192.168.1.101:554/cam/realmonitor?channel=1&subtype=01
      - ffmpeg:Driveway_sub#audio=opus    
    driveway:
      - rtsp://user:pass@192.168.1.101:554/cam/realmonitor?channel=1&subtype=0#backchannel=0
      - ffmpeg:Driveway#audio=opus
    driveway_sub:
      - rtsp://user:pass@192.168.1.101:554/cam/realmonitor?channel=1&subtype=01#backchannel=0
      - ffmpeg:Driveway_sub#audio=opus
    backyard:
      - rtsp://user:pass@192.168.1.113/live#video=h264/
      - ffmpeg:rtsp://user:pass@192.168.1.113/live/ch00_1
    flowerbeds:
      - rtsp://192.168.1.212:8554/1080p?mp4
      - ffmpeg:rtsp://192.168.1.212:8554/1080p?mp4#audio=opus
    frontdoor:
      - rtsp://user:pass@192.168.1.105:554/cam/realmonitor?channel=1&subtype=0 # Connect to fullres
      - ffmpeg:rtsp://user:pass@192.168.1.105:554/cam/realmonitor?channel=1&subtype=0#audio=opus
    frontdoor_sub:
      - rtsp://user:pass@192.168.1.105:554/cam/realmonitor?channel=1&subtype=1 # Connect to substream for lower bandwidth
      - ffmpeg:rtsp://user:pass@192.168.1.105:554/cam/realmonitor?channel=1&subtype=0#audio=opus
    room:
      - rtsp://user:pass@192.168.1.108/live#video=h264
  webrtc:
    candidates:
      - 192.168.1.34:8555
      - stun:8555

cameras:
  room:
    ffmpeg:
      inputs:
        - path: rtsp://192.168.1.34:8554/room
          roles:
            - record
    record:
      enabled: true
      retain:
        days: 3
        mode: all
    snapshots:
      enabled: false
    detect:
      enabled: false

  frontdoor:
    ffmpeg:
      inputs:
        - path: rtsp://192.168.1.34:8554/frontdoor_sub
          roles:
            - detect
        - path: rtsp://192.168.1.34:8554/frontdoor
          roles:
            - record
    detect:
      width: 720
      height: 576
      fps: 5
    record:
      enabled: true
      retain:
        days: 5
        mode: all
      alerts:
        retain:
          days: 5
      detections:
        retain:
          days: 5
    snapshots:
      enabled: true
      retain:
        default: 5
      required_zones:
        - frontporch_zone
    objects:
      track:
        - person
        - face
        - dog
        - cat
      filters:
        dog:
          min_score: 0.7
          threshold: 0.7
          mask: 0.46,0.85,0.624,0.865,0.627,0.999,0.435,0.998
        cat:
          min_score: 0.7
          threshold: 0.7
          mask: 0.46,0.851,0.429,0.994,0.642,1,0.637,0.852
        person:
          mask: 0.424,0.654,0.427,0.764,0.508,0.768,0.509,0.658
    motion:
      threshold: 40
      mask: 0,0,1,0,1,0.576,0.694,0.6,0.463,0.611,0.246,0.61,0,0.597
      contour_area: 25
      improve_contrast: 'true'
    review:
      alerts:
        labels:
          - person
          - face
        required_zones:
          - frontporch_zone
      detections:
        labels:
          - cat
          - dog
        required_zones:
          - frontporch_zone
    zones:
      frontporch_zone:
        coordinates: 
          0.428,1,1,1,1,0.769,1,0.61,0.71,0.639,0.418,0.657,0.179,0.646,0.174,0.746,0,0.869,0,1

        inertia: 3

  driveway:
    ffmpeg:
      inputs:
        - path: rtsp://192.168.1.34:8554/driveway_sub
          roles:
            - detect
            - audio
        - path: rtsp://192.168.1.34:8554/driveway
          roles:
            - record
    detect:
      width: 640
      height: 480
      fps: 5
    record:
      enabled: true
      retain:
        days: 5
        mode: all
      alerts:
        retain:
          days: 5
      detections:
        retain:
          days: 5
    snapshots:
      enabled: true
      retain:
        default: 5
      required_zones:
        - driveway_zone
        - driveway_grass
    objects:
      track:
        - person
        - face
        - dog
        - cat
      filters:
        dog:
          min_score: 0.7
          threshold: 0.7
        cat:
          min_score: 0.7
          threshold: 0.7
    motion:
      threshold: 25
      mask:
        - 0,0.638,0,0.3,0,0,0.403,0,1,0,1,0.276,0.942,0.226,0.686,0.195,0.389,0.207,0.239,0.24,0.144,0.272,0.064,0.323,0.117,0.511
        - 0.698,1,0.953,1,0.944,0.759,0.662,0.688,0.462,0.649,0.189,0.988
    review:
      alerts:
        labels:
          - person
          - face
        required_zones:
          - driveway_zone
          - driveway_grass
      detections:
        labels:
          - cat
          - dog
        required_zones:
          - driveway_zone
          - driveway_grass
    zones:
      driveway_zone:
        coordinates: 
          0,1,0.166,1,0.572,1,1,1,1,0.573,0.912,0.298,0.881,0.264,0.745,0.251,0.487,0.238,0.444,0.264,0.252,0.438,0,0.7
        inertia: 3
      driveway_grass:
        coordinates: 0.119,0.312,0.183,0.281,0.314,0.25,0.445,0.236,0.249,0.416,0.018,0.391
        inertia: 3
    audio:
      # Optional: Enable audio events (default: shown below)
      enabled: true
      # Optional: Configure the amount of seconds without detected audio to end the event (default: shown below)
      max_not_heard: 20
      # Optional: Configure the min rms volume required to run audio detection (default: shown below)
      # As a rule of thumb:
      #  - 200 - high sensitivity
      #  - 500 - medium sensitivity
      #  - 1000 - low sensitivity
      min_volume: 500
      # Optional: Types of audio to listen for (default: shown below)
      listen:
        - bark
        - fire_alarm
        - scream
        - yell
      # Optional: Filters to configure detection.
      filters:
        # Label that matches label in listen config.
        speech:
          # Minimum score that triggers an audio event (default: shown below)
          threshold: 0.5

  backyard:
    ffmpeg:
      inputs:
        - path: rtsp://192.168.1.34:8554/backyard
          roles:
            - record
            - detect
    detect:
      width: 1920
      height: 1080
      fps: 5
    record:
      enabled: true
      retain:
        days: 5
        mode: all
      alerts:
        retain:
          days: 5
      detections:
        retain:
          days: 5
    snapshots:
      enabled: true
      retain:
        default: 5
      required_zones:
        - backyard_zone
    objects:
      track:
        - person
        - face
        - cat
        - dog
      filters:
        person: {}
    motion:
      mask:
        - 0,0.247,0.047,0.2,0.226,0.111,0.387,0.07,0.507,0.052,0.588,0.057,0.65,0.067,0.766,0.087,0.908,0.129,1,0.205,1,0,0.429,0,0,0
        - 0.729,0.981,0.979,0.981,0.974,0.935,0.723,0.935
        - 0.811,1,0.805,0.973,0.615,0.793,0.447,1
      threshold: 40
      contour_area: 30
      improve_contrast: 'true'
    zones:
      backyard_zone:
        coordinates: 
          0.045,0.473,0.043,1,0.44,0.995,0.614,0.78,0.712,0.871,0.83,0.617,0.906,0.388,0.919,0.347,0.934,0.262,0.836,0.216,0.712,0.158,0.658,0.14,0.514,0.176,0.401,0.21,0.256,0.263,0.262,0.337
        objects: person
        inertia: 3
        loitering_time: 0
    review:
      alerts:
        labels:
          - person
          - face
        required_zones:
          - backyard_zone
      detections:
        labels:
          - cat
          - dog
        required_zones:
          - backyard_zone
  flowerbeds:
    ffmpeg:
      inputs:
        - path: rtsp://192.168.1.34:8554/flowerbeds
          roles:
            - detect
            - record
    detect:
      width: 1920
      height: 1080
      fps: 5
    record:
      enabled: true
      retain:
        days: 5
        mode: all
      alerts:
        retain:
          days: 5
      detections:
        retain:
          days: 5
    snapshots:
      enabled: true
      retain:
        default: 5
      required_zones:
        - flowerbed_zone
        - flowerbed_grass
    objects:
      track:
        - dog
        - cat
        - person
        - face
      filters:
        dog:
          min_score: 0.7
          threshold: 0.7
        cat:
          min_score: 0.7
          threshold: 0.7
        person:
          mask:
            - 0.811,0.784,0.853,0.754,0.593,0.081,0.55,0.066
            - 0.432,1,0.773,0.774,0.805,0.853,0.639,1
      mask: 0.352,0.619,0.316,0.071,0.271,0.07,0.271,0.631
    motion:
      threshold: 35
      mask:
        - 0.976,0.938,0.969,0.981,0.722,0.981,0.719,0.934
        - 1,0,0.782,0,0,0,0,1,0.225,1,0.254,0.95,0.239,0.739,0.23,0.535,0.231,0.285,0.243,0.097,0.204,0.094,0.21,0.01,0.43,0.038,0.592,0.09,0.744,0.168,0.829,0.217,0.871,0.299,0.957,0.49,1,0.621
      improve_contrast: 'true'
      contour_area: 20
    review:
      alerts:
        labels:
          - person
          - face
        required_zones:
          - flowerbed_zone
          - flowerbed_grass
      detections:
        labels:
          - cat
          - dog
        required_zones:
          - flowerbed_zone
          - flowerbed_grass
    zones:
      flowerbed_zone:
        coordinates: 
          0.216,0.018,0.209,0.087,0.248,0.087,0.234,0.302,0.233,0.523,0.239,0.668,0.25,0.868,0.258,0.951,0.227,1,0.322,1,0.738,1,0.858,0.875,0.726,0.478,0.639,0.257,0.57,0.091,0.433,0.047
        inertia: 3
        loitering_time: 0
      flowerbed_grass:
        coordinates: 
          0.579,0.094,0.732,0.47,0.866,0.875,1,0.699,0.991,0.617,0.927,0.444,0.816,0.217,0.722,0.165
        inertia: 3
        loitering_time: 0
version: 0.15-0
camera_groups:
  Frontyard:
    order: 1
    icon: LuCar
    cameras:
      - driveway
      - flowerbeds
      - frontdoor
  Backyard:
    order: 2
    icon: LuFlower2
    cameras: backyard
  Sutton:
    order: 3
    icon: LuBaby
    cameras: room

```

**Reply by NickM-27** [reply]
to confirm, does this issue happen when running the `small` model?

**Reply by coldkevin** [reply]
changing to small does work and the semantic search starts to index but my CPU spikes obviously and im trying to keep as much load off the cpu if possible so i was trying to figure out how to use the nvdia gpu for semantic search. Do I use the tensor image and define the gpu as another detector?

**Reply by NickM-27** [reply]
just use the tensorrt image and set the model size to large, it is all automatic

**Reply by coldkevin** [reply]
It actually just finished indexing with the cpu and things have settled down, ill take this for a spin. is the load only when new snapshots are generated? I tried a few searches and my cpu didnt really budge so thats nice. in other news, the search works really well and I thought this was hilarious. 
![image](https://github.com/user-attachments/assets/d99772a0-83aa-4b8a-aa6e-4c38138758cd)

**Reply by NickM-27** [reply]
yes, it is just a spike when it is running the embeddings inferences

**Comment by ivanjx** [comment]
is this specifically for embeddings? because in my case i already disabled it and frigate keeps crashing. im on latest dev.

**Comment by hawkeye217** [comment, marked as answer]
This should be fixed in https://github.com/blakeblackshear/frigate/pull/14887
